Twig/Extension/SonataAdminExtension.php

```diff
@@ -21,6 +21,9 @@
 /**
  * Class SonataAdminExtension.
  *
+ * The output and getValueFromFieldDescription methods are public so that they
+ * can be tested, but should not be relied on.
+ *
```

Comment:
I don't think `@internal` should be used. This class should not be extended, but can be used by developer outside the internal API.